service_catalog/views/request.py

```diff
@@ -72,6 +72,30 @@ def request_comment(request, request_id):
     return render(request, "service_catalog/common/request-comment.html", context)
 
 
+@login_required
+@permission_required_or_403('service_catalog.comment_request', (Request, 'id', 'request_id'))
```

Comment:
test to be added to check that only the owner or admin can edit the message.
Same for support.

Reply:
done